cli/cmd/set.go

```diff
@@ -265,6 +265,9 @@ type setDoer struct {
 }
 
 func (d *setDoer) Do(ctx context.Context) error {
+	// 30s hard-coded in aTLS + 5s for Coordinator storage sync.
+	ctx, cancel := context.WithTimeout(ctx, 35*time.Second)
```

Comment:
If we want a greater timeout, then we might want to define those in a central place. Also I think 35 might be a bit on the low side, since we set the minimal timeout to 30 seconds, it might very well be higher.